src/functionsDeployHelper.ts

```diff
@@ -6,10 +6,12 @@ import * as logger from "./logger";
 import * as track from "./track";
 import * as utils from "./utils";
 import * as cloudfunctions from "./gcp/cloudfunctions";
+import { Job } from "./gcp/cloudscheduler";
 import * as pollOperations from "./pollOperations";
 import { CloudFunctionTrigger } from "./deploy/functions/deploymentPlanner";
+import Queue from "./throttler/queue";
 
-// TODO: Get rid of this when switching to use operation-poller.
+// TODO(joehan): Get rid of this once we refactor functions-delete.js
```

Comment:
This file has a mix of random helper functions, and chaff from the old deployment code that I can't quite delete yet because its used on functions-delete and other code paths. I'm gonna make another PR later to clean up the old functions, and move the still relevant functions into a less generically named file inside of the src/deploy/functions directory.